sagify/commands/initialize.py

```diff
@@ -101,6 +105,8 @@ def init(dir):
     """
     logger.info(ASCII_LOGO)
 
+    ecr_repository_name = ask_for_ecr_repository_name()
```

Comment:
I think it's worth reconsidering where to best put this config. I feel like most users will be fine with the default repo name, so I wonder if there a better place than the init script.

However, I do see that once this option is set, it will not need to be changed frequently (if at all)

Reply:
yeah, I don't think this will change frequently.

Reply:
@berlinquin it LGTM in general. Do you have any suggestions or comments?